packages/database-persistence-layer/src/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import { store as preferencesStore } from '@wordpress/preferences';
+
+/**
+ * Internal dependencies
+ */
+import create from './create';
+
+export { create };
+
+export function __experimentalConfigureDatabasePersistenceLayer( options ) {
+	const persistenceLayer = create( { options } );
+	dispatch( preferencesStore ).setPersistenceLayer( persistenceLayer );
```

Comment:
Initially I wanted this code to be in the inline script, but I couldn't figure out how to make an inline script have multiple dependencies.

Exposing one function from the package seemed to be the easiest option, but that means adding `data` and `preferences` as dependencies to the package, which otherwise wouldn't have been necessary.

Reply:
I think in my mind it would have been something like that:

```php
wp_add_dependency_to_script( 'wp-preferences', 'wp-database-persistence-layer' );
wp_add_inline_script( 'wp-preferences', 'wp.data.dispatch( wp.preferences.store ).setPersistenceLayer( wp.databasePersistenceLayer.create() );', 'after' );
```

In the above I'm adding `wp-database-persistence-layer` as a dependency to `wp-preferences` but it can also be done at registration for `wp-preferences` in the script loader.